app/Http/Controllers/Api/ReportbackController.php

```diff
@@ -43,7 +43,7 @@ public function index(Request $request)
         // 2. Only return approved Posts
         // 3. Select all the fields that we will be using
         $query = $query->join('signups', 'signups.id', '=', 'posts.signup_id')
-            ->where('status', '=', 'approved')
+            ->where('status', '=', 'accepted')
```

Comment:
Just a thought for the future – it might be nice to make a set of constants that we could import elsewhere when needed so we don't accidentally make typos (especially for ones like this where it's so easy to do!)